programs/cross-chain-escrow-src/tests/test_cross_chain_escrow_src_native.rs

```diff
@@ -74,6 +74,23 @@ mod test_native_src {
         );
     }
 
+    #[test_context(TestState)]
+    #[tokio::test]
+    async fn test_escrow_creation_fails_with_unwhitelisted_resolver(test_state: &mut TestState) {
```

Comment:
It seems this test isn’t needed, since there’s already a similar one in the SPL section.